docs/sql-migration-guide.md

```diff
@@ -43,7 +43,7 @@ license: |
 
   - The `ADD JAR` command previously returned a result set with the single value 0. It now returns an empty result set.
 
-  - In Spark version 2.4 and earlier, users can create map values with map type key via built-in function like `CreateMap`, `MapFromArrays`, etc. Since Spark 3.0, it's not allowed to create map values with map type key with these built-in functions. Users can still read map values with map type key from data source or Java/Scala collections, though they are not very useful.
+  - In Spark version 2.4 and earlier, users can create map values with map type key via built-in function such as `CreateMap`, `MapFromArrays`, etc. Since Spark 3.0, it's not allowed to create map values with map type key with these built-in functions. Users can use array<struct<key, value>> as map keys instead as a workaround. In addition, users can still read map values with map type key from data source or Java/Scala collections, though it is discouraged.
```

Comment:
Could we give a very simple example, how to use array<struct<key, value>> as map keys as a workaround? Also, do we have such a test case?